src/10-webpage-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,311 @@
+---
+date: 2016-02-23
+id: 10-webpage-publishing-with-ipfs
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "10-webpage-publishing-with-ipfs", link: "./" }
+tags: publishing, web, app, publish
+title: Static Web publishing, simple and distributed with IPFS
+author: David Dias
+collection: posts
+---
+
+> - Don't you hate it when your links break?
+> - Don't you hate it when your webhost gets attacked and your websites go down?
+> - Don't you hate managing servers just to self-host a webpage?
+> - Don't you hate it when you switch cloud providers and all the data links change?
+> - Don't you hate it when people relying on your content get broken embeds?
+
+> NOTE: I think it'd be better to start stronger than rhetorical questions. For instance:    
+> The web is broken. Links break; webhosts get attacked, and websites go down (often permanently). As a web developer, you have to by a sysadmin just to host a webpage. If you switch cloud providers, links break. Even with a fully functional website, more issues might pop up; link rot, or broken embeds for third parties serving your content. It's 2016, and not all is as it should be.
```

Comment:
We're going to have to go with one of these, or the other. The questions, or the paragraph.